app/components/views/GetStartedPage/CreateWallet/CreateWalletForm/ExistingSeed/Form.js

```diff
@@ -80,9 +89,14 @@ class ExistingSeedForm extends React.Component{
           </div>
           {seedType == "words" && Array.isArray(seedWords) ?
             <div className="seedArea">
-              {!this.state.showPasteWarning ? null : <div className="orange-warning">
+              {this.state.showPasteWarning &&
+              <div className="orange-warning">
                 <T id="confirmSeed.warnings.pasteExistingSeed" m="*Please make sure you also have a physical, written down copy of your seed." />
               </div>}
+              {this.state.showPasteError &&
+              <div className="red-error">
```

Comment:
Could you change this from `red-error` to `error`?

Reply:
sure thing